NextcloudTalk/ContactsSearchResultTableViewContoller.swift

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright (c) 2024 Ivan Sein <ivan@nextcloud.com>
+//
+// Author Ivan Sein <ivan@nextcloud.com>
+//
+// GNU GPL version 3 or any later version
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+@objcMembers class ContactsSearchResultTableViewController: UITableViewController {
+
+    var indexes: [String] = []
+    var contacts: [String: [NCUser]] = [:]
+
+    let tableBackgroundView = PlaceholderView()
```

Comment:
```suggestion
    let tableBackgroundView = PlaceholderView(for: .insetGrouped)!
```


https://github.com/nextcloud/talk-ios/assets/1580193/a1a31f82-9e69-4310-b6ff-c07e83ba73ed